applications/RANSApplication/custom_strategies/algebraic_flux_corrected_scalar_steady_scheme.h

```diff
@@ -0,0 +1,567 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:  Suneth Warnakulasuriya
+//                 Michael Andre, https://github.com/msandre
+//
+
+#if !defined(KRATOS_ALGEBRAIC_FLUX_CORRECTED_SCALAR_STEADY_SCHEME)
+#define KRATOS_ALGEBRAIC_FLUX_CORRECTED_SCALAR_STEADY_SCHEME
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "solving_strategies/schemes/scheme.h"
+#include "utilities/openmp_utils.h"
```

Comment:
I'd avoid adding this one in new features if possible.

Reply:
I dont understand? :/